docs/sources/migration-guide/migrating-from-single-zone-with-helm.md

```diff
@@ -0,0 +1,715 @@
+         writePath: true
+         readPath: true
+         excludeDefaultZone: true
+
+   rollout_operator:
+     enabled: true
+   ```
+
+   [//]: # "ingester-step5"
+
+1. Upgrade the installation with the `helm` command and make sure to provide the flag `-f migrate.yaml` as the last flag.
+
+   In this step the flag `-distributor.excluded-zones=zone-default` is added to distributors and rulers.
+
+1. Wait until all distributors and rulers have restarted and are ready.
+
+1. Scale down non zone aware ingesters to 0.
+
+   Replace the contents of the `migrate.yaml` file with:
+
+   ```yaml
+   ingester:
+     zoneAwareReplication:
+       enabled: true
+       migration:
+         enabled: true
+         writePath: true
+         readPath: true
+         excludeDefaultZone: true
+         scaleDownDefaultZone: true
+
+   rollout_operator:
+     enabled: true
+   ```
+
+   [//]: # "ingester-step6"
+
+1. Upgrade the installation with the `helm` command and make sure to provide the flag `-f migrate.yaml` as the last flag.
+
+1. Wait until all non zone aware ingesters are terminated.
+
+1. Delete the default zone.
+
+   **Merge** the following values into your custom Helm values file:
+
+   ```yaml
+   ingester:
+     zoneAwareReplication:
+       enabled: true
+
+   rollout_operator:
+     enabled: true
+   ```
+
+   [//]: # "ingester-step7"
+
+   These values are actually the default, which means that removing the values `ingester.zoneAwareReplication.enabled` and `rollout_operator.enabled` from your `custom.yaml` is also a valid step.
+
+1. Upgrade the installation with the `helm` command using your regular command line flags.
+
+1. Wait at least 3 hours.
+
+   The 3 hours is calculated from `blocks_storage.tsdb.block_ranges_period` + `blocks_storage.tsdb.head_compaction_idle_timeout` Grafana Mimir parameters to give enough time for ingesters to remove stale series from memory. Stale series will be there due to series being moved between ingesters.
+
+1. If you are using [shuffle sharding]({{< relref "../operators-guide/configure/configuring-shuffle-sharding" >}}):
+
+   1. Wait an extra 12 hours.
+
+      The 12 hours is calculated from the `querier.query_store_after` Grafana Mimir parameter. After this time, no series are stored outside their dedicated shard, meaning that shuffle sharding on the read path can be safely enabled.
+
+   1. Remove these values from your configuration:
+
+      ```yaml
+      querier:
+        extraArgs:
+          "querier.shuffle-sharding-ingesters-enabled": "false"
+      ruler:
+        extraArgs:
+          "querier.shuffle-sharding-ingesters-enabled": "false"
+      ```
+
+   1. Upgrade the installation with the `helm` command using your regular command line flags.
+
+   1. Wait until queriers and rulers have restarted and are ready.
+
+1. Undo the doubling of series limits done in the first step.
+
+1. Upgrade the installation with the `helm` command using your regular command line flags.
```

Comment:
so I am writing the getting to production guide. I wanted to get a single snippet of yaml that enables zone-awareness on a fresh installation. Is this available anywhere? I think having it in values.yaml would be most useful (like a commented out YAML block)

I found this further up in the article and I think I should just toggle `enabled`, but I can't be sure
```yaml
   ingester:
     zoneAwareReplication:
       enabled: false # Do not turn on zone awareness without migration because of potential data loss
       topologyKey: "kubernetes.io/hostname" # Triggers creating anti-affinity rules
       zones:
         - name: zone-a
           nodeSelector:
             topology.kubernetes.io/zone: zone-a
         - name: zone-b
           nodeSelector:
             topology.kubernetes.io/zone: zone-b
         - name: zone-c
           nodeSelector:
             topology.kubernetes.io/zone: zone-c
```

Reply:
Yes, toggle enabled to true (although for ingesters and store-gateways it will be true by default).
I'll revise the inline documentation in values.yaml for clarity.

Reply:
done, please check